faiss/utils/hamming.cpp

```diff
@@ -516,10 +538,11 @@ void hamming_range_search(
         size_t nb,
         int radius,
         size_t code_size,
-        RangeSearchResult* result) {
+        RangeSearchResult* result,
+        const IDSelector* sel = nullptr) {
```

Comment:
There is currently a compilation error in the CI. The default argument value is only necessary in the header file, not here. Thanks!